### PR TITLE
remove shellquote for passwords

### PR DESCRIPTION
**Pull request by CommanderK5**

### CODE REVIEW

**Review by shiroyasha — APPROVED** [review]

**Comment by shiroyasha** [comment]
Looks great 🙇 

Thanks for executing this.

File: pkg/executors/docker_compose_executor.go [hunk 1]
```diff
@@ -262,7 +262,7 @@ func (e *DockerComposeExecutor) injectImagePullSecretsForDockerHub(envVars []api
 			return 1
 		}
 
-		envs = append(envs, fmt.Sprintf("%s=%s", name, ShellQuote(string(value))))
```
Comment (shiroyasha) [inline pkg/executors/docker_compose_executor.go:265]: Do we also need to remove the `string(...)` part? 🤔

Comment (CommanderK5) [reply pkg/executors/docker_compose_executor.go:265]: I think we can remove it, based on the api spec
```
type EnvVar struct {
	Name  string `json:"name" yaml:"name"`
	Value string `json:"value" yaml:"value"`
}
```

Comment (shiroyasha) [reply pkg/executors/docker_compose_executor.go:265]: Please also check this line https://github.com/semaphoreci/agent/pull/89/files/cd6fde09ee6b90aae70845fc29fe104e964eab27#diff-d195318a5e14bce2a0703d102fc4e4ff776fda21bed0d91cbd05da6cc8724614R258. We are decoding the value there.

**Pull request merged**